lua/autorun/sh_cami.lua

```diff
@@ -388,10 +238,31 @@ local defaultAccessHandler = {["CAMI.PlayerHasAccess"] =
         callback(false, "No information available.")
     end
 }
+
+--- @class CAMI_ACCESS_EXTRA_INFO
+--- @field Fallback "'user'" | "'admin'" | "'superadmin'" @Fallback status for if the privilege doesn't exist. Defaults to `admin`.
+--- @field IgnoreImmunity boolean @Ignore any immunity mechanisms an admin mod might have.
+--- @field CommandArguments table @Extra arguments that were given to the privilege command.
+
+--- Checks if a player has access to a privilege
+--- (and optionally can execute it on targetPly)
+---
+--- This function is designed to be asynchronous but will be invoked
+---  synchronously if no callback is passed.
+---
+--- ⚠ **Warning**: If the currently installed admin mod does not support
+---                 synchronous queries, this function will throw an error!
+--- @param actorPly GPlayer @The player to query
+--- @param privilegeName string @The privilege to query
+--- @param callback fun(hasAccess: boolean, reason: string|nil) @Callback to receive the answer, or nil for synchronous
+--- @param targetPly GPlayer | nil @Optional - target for if the privilege effects another player (eg kick/ban)
+--- @param extraInfoTbl CAMI_ACCESS_EXTRA_INFO | nil @Table of extra information for the admin mod
+--- @return boolean | nil @Synchronous only - if the player has the privilege
+--- @return string | nil @Synchronous only - optional reason from admin mod
 function CAMI.PlayerHasAccess(actorPly, privilegeName, callback, targetPly,
 extraInfoTbl)
     local hasAccess, reason = nil, nil
-    local callback_ = callback or function(hA, r) hasAccess, reason = hA, r end
+	local callback_ = callback or function(hA, r) hasAccess, reason = hA, r end
```

Comment:
```suggestion
    local callback_ = callback or function(hA, r) hasAccess, reason = hA, r end
```
Inconsistent use of tabs and spaces 🙃

Reply:
how the fuck did that happen